nmstatectl/nmstatectl.py

```diff
@@ -157,6 +158,17 @@ def setup_subcommand_show(subparsers):
     )
 
 
+def setup_subcommand_displayversion(subparsers):
+    parser_display_version = subparsers.add_parser(
+        "version",
```

Comment:
It is not possible to rename this to "--version"? Just asking I didn't try it.

Reply:
I get a message saying it expects an argument, I can definitely change it but I don't quite know how to play around that arg requirement.

Reply:
I get an error like this:
``` bash
[root@16e920d9c918 nmstate-workspace]# nmstatectl --version
usage: nmstatectl [-h] {commit,edit,rollback,set,show,--version} ...
nmstatectl: error: unrecognized arguments: --version
```

Reply:
I see, let's keep it as `nmstatectl version`. @tyll What do you think?

Reply:
For `--version` it needs something like this:

```
parser.add_argument("--version")
...
if args.version:
   print version
else:
    return args.func(args)
```

`nmstatectl version` makes probably more sense but I would assume everyone would expect `nmstatectl --version` to work, too.

Reply:
Yes, it would be nice to have both. Thanks!

Reply:
I'll try my best to figure out how to do it with ```nmstatectl --version```, since that's also the way I'd expect things to work. I was reading argparse documentation but the way the main function is structured makes it harder for me follow. Are we sure this is something that can be implemented within ```nmstatectl.py``` ?

Reply:
Did you try the approach I outlined in https://github.com/nmstate/nmstate/pull/920#discussion_r390406218?

Reply:
@tyll Yep, finally made it work, thanks. Just had to check documentation to make it not require extra arguments past ```--version```. Will update the PR with the changes and the integration test I wrote.